sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/BasicWriteStatsTracker.scala

```diff
@@ -151,7 +151,7 @@ class BasicWriteTaskStatsTracker(hadoopConf: Configuration)
     }
   }
 
-  override def newRow(row: InternalRow): Unit = {
+  override def newRow(filePath: String, row: InternalRow): Unit = {
```

Comment:
From Apache Side, this is no-op parameter and there is no test coverage in this PR. Do you think we can have a sample custom `WriteTaskStatsTracker` test case to prevent a future regression, @cloud-fan ?
> To not break some custom WriteTaskStatsTracker implementations.